nwb_conversion_tools/datainterfaces/ecephys/neuroscope/neuroscopedatainterface.py

```diff
@@ -18,32 +19,37 @@
     from .neuroscope_utils import get_xml_file_path, get_channel_groups, get_shank_channels
 
     HAVE_LXML = True
+
 except ImportError:
     HAVE_LXML = False
 INSTALL_MESSAGE = "Please install lxml to use this interface!"
 
+SpikeInterfaceRecording = Union[se.RecordingExtractor, BaseRecording]
```

Comment:
Might as well import this from here: https://github.com/catalystneuro/nwb-conversion-tools/blob/update_neuroscope_interface_to_spike_interface/nwb_conversion_tools/tools/spikeinterface/spikeinterface.py#L24

Rather then re-define it

Reply:
Great.
I turns out that I no longer need it in this specific instance. For the future though, maybe we should consolidate these imports in the `utils/typing` as we have some other types there?

Reply:
They are not, however, `utils` since they require external dependencies. They belong in tools, and the location in the `tools.spikeinterface` seems most appropriate.

Reply:
Fair enough.